prescriptions/_containers/s2i_thoth_ubi8_py38/quay_security.yaml

```diff
@@ -5249,7 +5249,7 @@ units:
       - type: WARNING
         message: >-
           Found vulnerability 'pyup.io-43975 (CVE-2021-33503)' in the base image used
-        link: 
+        link:
```

Comment:
This will not pass schema validation - the `link` cannot be empty. I think we have a bug in the implementation.